Remotion/Web/Core/Res/Themes/NovaViso/HTML/Common.css

```diff
@@ -0,0 +1,110 @@
+:root {
+    /*colors*/
+    --primary: #0C6594;
+    --highlight: #DEF3FE;
+    /*not needed?*/
+    --gray: #C4C4C4;
+    /*not needed?*/
+    --light-gray: #E1E1E1;
+    --menu-background: #F9F9F9;
+    --hover: #218FC9;
+    --border: #979797;
+    --yellow: #DFAF00;
+    --red: #DA2222;
+    --green: #4AD439;
+    --focus: #23ADF8;
+    --header: #F2F2F2;
+
+    --dropdown-scrollbar-track: #E3E3E3;
+    --scrollbar-track: #D3D3D3D3;
+    --background: #FFFFFF;
+
+    /*remotion-themed*/
+    --remotion-themed-border: thin solid var(--border);
+    --remotion-themed-border-focus: thin solid var(--focus);
+    --remotion-themed-border-radius: 0.125em;
+    --remotion-themed-border-hover: thin solid var(--primary);
+    --remotion-themed-dropdown-scrollbar-border-radius: 0.6250em;
+    --remotion-themed-dropdown-scrollbar-width: 0.6250em;
+    --remotion-themed-scrollbar-width: 0.8750em;
+}
+
+input.remotion-themed,
+textarea.remotion-themed,
+select.remotion-themed,
+.remotion-themed input,
+.remotion-themed textarea,
+.remotion-themed select {
+    border: var(--remotion-themed-border);
+    border-radius: var(--remotion-themed-border-radius);
+}
+
+input.remotion-themed:hover,
+textarea.remotion-themed:hover,
+select.remotion-themed:hover,
+.remotion-themed input:hover,
+.remotion-themed textarea:hover,
+.remotion-themed select:hover {
+    border: var(--remotion-themed-border-hover);
+    outline-style: none;
+}
+
+input.remotion-themed:focus,
+textarea.remotion-themed:focus,
+select.remotion-themed:focus,
+.remotion-themed input:focus,
+.remotion-themed textarea:focus,
+.remotion-themed select:focus {
+    border: var(--remotion-themed-border-focus);
```

Comment:
For focus, we should try to use outline, not border. There's a special intention/semantics to it *) and I don't think the styleguide actually covers this yet. If not, we need a good default and a follow-up ticket (or a remark in the existing follow-up ticket)

There's also the aspect of the size change. If the focus should have a thicker outline, this can be done with css-outline, but not with border without changing the layout (I think)

Reply:
The problem with outline would be that the corners cannot be rounded.

Reply:
outline's a separate concept from border and used in addition to the border. The shape of the outline is not that much customizable, but very prominent. 

=> please make a follow-up ticket for the focus effect because this should be decided when we can actually see the completed theme in a real-world scenario.